geps/gep-2649/index.md

```diff
@@ -0,0 +1,744 @@
+
+#### Defaults interacting with other defaults for RetryOnPolicy, empty list in HTTPRoute
+||No default|Namespace default A|Gateway default A|HTTPRoute default A|
+|----|-----|-----|----|----|
+|No default|Empty list|Namespace default| Gateway default| HTTPRoute default A|
+|Namespace default B| Namespace default B| Namespace default<br />first created wins<br />otherwise first alphabetically | Gateway default A | HTTPRoute default A|
+|Gateway default B| Gateway default B| Gateway default B| Gateway default<br />first created wins<br />otherwise first alphabetically | HTTPRoute default A|
+|HTTPRoute default B| HTTPRoute default B| HTTPRoute default B| HTTPRoute default B| HTTPRoute default<br />first created wins<br />otherwise first alphabetically|
+
+
+Now, if the HTTPRoute _does_ specify a RetryPolicy,
+it's a bit easier, because we can basically disregard all defaults:
+
+#### Overrides interacting with defaults for RetryOnPolicy, value in HTTPRoute
+
+||None|Namespace override|Gateway override|HTTPRoute override|
+|----|-----|-----|----|----|
+|No default| Value in HTTPRoute|Namespace override| Gateway override | HTTPRoute override|
+|Namespace default|  Value in HTTPRoute| Namespace override | Gateway override | HTTPRoute override |
+|Gateway default|  Value in HTTPRoute | Namespace override | Gateway override | HTTPRoute override |
+|HTTPRoute default| Value in HTTPRoute | Namespace override | Gateway override | HTTPRoute override|
+
+#### Overrides interacting with other overrides for RetryOnPolicy, value in HTTPRoute
+||No override|Namespace override A|Gateway override A|HTTPRoute override A|
+|----|-----|-----|----|----|
+|No override|Value in HTTPRoute|Namespace override A| Gateway override A| HTTPRoute override A|
+|Namespace override B| Namespace override B| Namespace override<br />first created wins<br />otherwise first alphabetically | Namespace override B| Namespace override B|
+|Gateway override B| Gateway override B| Namespace override A| Gateway override<br />first created wins<br />otherwise first alphabetically | Gateway override B|
+|HTTPRoute override B| HTTPRoute override B | Namespace override A| Gateway override A| HTTPRoute override<br />first created wins<br />otherwise first alphabetically|
+
+#### Defaults interacting with other defaults for RetryOnPolicy, value in HTTPRoute
+||No default|Namespace default A|Gateway default A|HTTPRoute default A|
+|----|-----|-----|----|----|
+|No default|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|
+|Namespace default B|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|
+|Gateway default B|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|
+|HTTPRoute default B|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|
+
+
+## User discoverability and status
```

Comment:
Why doesn't this use the same status as BackendTLSPolicy/direct policy attachment as a baseline? I think we could add more on top of that for inherited policies, but that seems like a good baseline for all policies.

Reply:
I was leaving this for a future discussion, in the interest of getting this in. I will include the existing status suggestion from Direct Policy Attachment as a SHOULD here.

Reply:
I didn't include it because it seems very likely that it could create fanout problems in the Inherited case (since a Policy may have many ancestors).

I'd like to plan on following up here once @guicassolato and the Kuadrant folks have more information.